### PR TITLE
Clean cs project and add nuget package information

### PR DESCRIPTION
**Pull request by ddubuc**
fix https://github.com/aneoconsulting/ArmoniK/issues/486

### CODE REVIEW

File: Common/ArmoniK.DevelopmentKit.Common/ArmoniK.DevelopmentKit.Common.csproj [hunk 1]
```diff
@@ -20,27 +20,65 @@
...
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
     <DebugType>Embedded</DebugType>
     <IncludeSymbols>true</IncludeSymbols>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
 
-  <Target Name="BeforeBuild">
-  <Message Importance="High" Text="This the framework sss $(TargetFramework)" />
-  </Target>
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
```
Comment (lemaitre-aneo) [inline Common/ArmoniK.DevelopmentKit.Common/ArmoniK.DevelopmentKit.Common.csproj:69]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#optimize

> The Optimize option is enabled by default for a Release build configuration.

Comment (ddubuc) [reply Common/ArmoniK.DevelopmentKit.Common/ArmoniK.DevelopmentKit.Common.csproj:69]: Good to know. I think VS add it when I switch to release once

File: Tests/Armonik.Windows.Tests/Program.cs [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+                              "ArmoniK.Samples.GridServer.Services");
+
+      taskOptions.Options.Add(AppsOptions.GridAppVersionKey,
+                              "1.0.0-700");
+
+      taskOptions.Options.Add(AppsOptions.GridAppNamespaceKey,
+                              "ArmoniK.Samples.GridServer.Client.Services");
+
+      taskOptions.Options.Add(AppsOptions.GridServiceNameKey,
+                              "ServiceContainer");
```
Comment (dbrasseur-aneo) [inline Tests/Armonik.Windows.Tests/Program.cs:167]: Use the proper fields in the taskoptions

File: Tests/Armonik.Windows.Tests/Program.cs [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+      });
+      logger_ = factory.CreateLogger<Program>();
+      logger_.LogInformation("Starting to execute gridServer");
+
+
+
+      var taskOptions = InitializeSimpleTaskOptions();
+
+      // var props = new Properties(taskOptions, configuration_.GetSection("Grpc")["EndPoint"], 5001, sslValidation: false);
+      var props = new Properties(taskOptions, "https://ANEO-SB2-8454-wsl.local:5001", 5001, sslValidation: false);
```
Comment (dbrasseur-aneo) [inline Tests/Armonik.Windows.Tests/Program.cs:77]: Should this really be in the code ?

File: Tests/Armonik.Windows.Tests/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    ],
+    "Properties": {
+      "Application": "ArmoniK.Samples.SymphonyClient"
+    }
+  },
+  "Components": {
+  },
+
+  "Grpc": {
+    "EndPoint": "http://ANEO-SB2-8454-wsl.local:5001"
```
Comment (dbrasseur-aneo) [inline Tests/Armonik.Windows.Tests/appsettings.json:45]: Should this be the default ?

File: WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/Directory.Build.props [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: WorkerApi/tests/WorkerApiTests/Directory.Build.props [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline WorkerApi/tests/WorkerApiTests/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: WorkerApi/ArmoniK.DevelopmentKit.WorkerApi.Common/Directory.Build.props [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline WorkerApi/ArmoniK.DevelopmentKit.WorkerApi.Common/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: UnifiedApi/Worker/Directory.Build.props [hunk 8]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline UnifiedApi/Worker/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: UnifiedApi/Client/Directory.Build.props [hunk 9]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline UnifiedApi/Client/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: Tests/EndToEnd.Tests/Directory.Build.props [hunk 10]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline Tests/EndToEnd.Tests/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: Tests/Armonik.Windows.Tests/Properties/AssemblyInfo.cs [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+// Les informations générales relatives à un assembly dépendent de
+// l'ensemble d'attributs suivant. Changez les valeurs de ces attributs pour modifier les informations
+// associées à un assembly.
+[assembly: AssemblyTitle("Armonik.Windows.Tests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Armonik.Windows.Tests")]
+[assembly: AssemblyCopyright("Copyright ©  2022")]
```
Comment (dbrasseur-aneo) [inline Tests/Armonik.Windows.Tests/Properties/AssemblyInfo.cs:13]:
```suggestion
[assembly: AssemblyCopyright("Copyright © ANEO 2022")]
```

File: SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi/Directory.Build.props [hunk 12]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi.Client/Directory.Build.props [hunk 13]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi.Client/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: DataSynapseApi/ArmoniK.DevelopmentKit.GridServer/Directory.Build.props [hunk 14]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline DataSynapseApi/ArmoniK.DevelopmentKit.GridServer/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props [hunk 15]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: Common/ArmoniK.DevelopmentKit.Common/Directory.Build.props [hunk 16]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline Common/ArmoniK.DevelopmentKit.Common/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/Directory.Build.props [hunk 17]
```diff
@@ -0,0 +1,33 @@
+<Project>
+  <!--
+  This work allows to build on wsl and on windows with any conf²cts in object neither binary folders
```
Comment (dbrasseur-aneo) [inline Client/src/ArmoniK.DevelopmentKit.Client.Common/Directory.Build.props:3]:
```suggestion
  This work allows to build on wsl and on windows without any conflicts in either object or binary folders
```

**Review by dbrasseur-aneo — CHANGES_REQUESTED** [review]

File: Tests/Armonik.Windows.Tests/Program.cs (hunk 2 above)
Comment (ddubuc) [reply Tests/Armonik.Windows.Tests/Program.cs:167]: ok

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj [hunk 18]
```diff
@@ -1,37 +1,24 @@
...
-    <SignAssembly>True</SignAssembly>
-    <AssemblyOriginatorKeyFile>../../../kp.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="ArmoniK.Api.Client" Version="3.0.2-SNAPSHOT.32.1a7a62f" />
+ <ItemGroup>
+    <PackageReference Include="ArmoniK.Api.Client" Version="$(ArmonikAPIVersion)" />
     <PackageReference Include="Grpc.Core" Version="2.46.3" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.47.0" />
```
Comment (aneojgurhem) [inline Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj:11]: Why did it appear ?

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/Submitter/ClientServiceConnector.cs [hunk 19]
```diff
@@ -123,6 +124,26 @@ public class ClientServiceConnector
     }
 
 #if NET5_0_OR_GREATER
+    var defaultMethodConfig = new MethodConfig
```
Comment (aneojgurhem) [inline Client/src/ArmoniK.DevelopmentKit.Client.Common/Submitter/ClientServiceConnector.cs:127]: Does not seem to be linked to cleaning csproj or nuget packages

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj [hunk 20]
```diff
@@ -1,37 +1,24 @@
...
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <LangVersion>latest</LangVersion>
-    <SignAssembly>True</SignAssembly>
-    <AssemblyOriginatorKeyFile>../../../kp.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="ArmoniK.Api.Client" Version="3.0.2-SNAPSHOT.32.1a7a62f" />
+ <ItemGroup>
+    <PackageReference Include="ArmoniK.Api.Client" Version="$(ArmonikAPIVersion)" />
```
Comment (aneojgurhem) [inline Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj:9]: Is visual studio able to update this package ?

File: DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props [hunk 21]
```diff
@@ -0,0 +1,11 @@
+<Project>
```
Comment (aneojgurhem) [inline DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props:1]: Still too much props. RootProjetDir cannot be defined only once ? and import props in csproj ?

File: Tests/Armonik.Windows.Tests/App.config [hunk 22]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup>
```
Comment (aneojgurhem) [inline Tests/Armonik.Windows.Tests/App.config:3]: Is this file really necessary or only related to your machine ?

File: Tests/Armonik.Windows.Tests/Program.cs [hunk 23]
```diff
@@ -0,0 +1,255 @@
...
+
+      taskOptions.Options.Add(AppsOptions.GridAppVersionKey,
+                              "1.0.0-700");
+
+      taskOptions.Options.Add(AppsOptions.GridAppNamespaceKey,
+                              "ArmoniK.Samples.GridServer.Client.Services");
+
+      taskOptions.Options.Add(AppsOptions.GridServiceNameKey,
+                              "ServiceContainer");
+
```
Comment (aneojgurhem) [inline Tests/Armonik.Windows.Tests/Program.cs:167]: still not the proper fields and not the purpose of this pr

File: Tests/EndToEnd.Tests/Tests/HeavyMatrixCompute/HeavyMatrixCompute.cs [hunk 24]
```diff
@@ -0,0 +1,60 @@
+// This file is part of the ArmoniK project
+//
```
Comment (aneojgurhem) [inline Tests/EndToEnd.Tests/Tests/HeavyMatrixCompute/HeavyMatrixCompute.cs:2]: wrong purpose

File: WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/ArmoniK.DevelopmentKit.WorkerApi.csproj [hunk 25]
```diff
@@ -1,14 +1,22 @@
...
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="**\*.orig" />
+    <None Include="../../LICENSE" Pack="true" PackagePath="LICENSE.md" />
+    <None Include="../../armonik-logo.png" Pack="true" PackagePath="\" />
+    <None Include="../../README.md" Pack="true" PackagePath="\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0" PrivateAssets="All" />
```
Comment (aneojgurhem) [inline WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/ArmoniK.DevelopmentKit.WorkerApi.csproj:14]: This is added in one of the .props. Isn't it redundant ?

**Review by aneojgurhem — CHANGES_REQUESTED** [review]

File: Tests/Armonik.Windows.Tests/Program.cs (hunk 3 above)
Comment (ddubuc) [reply Tests/Armonik.Windows.Tests/Program.cs]: Removed

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj (hunk 20 above)
Comment (ddubuc) [reply Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj:9]: yes

File: DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props (hunk 21 above)
Comment (ddubuc) [reply DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Directory.Build.props:1]: Needed for relative path in each Common property in PropertyGroup

File: Tests/Armonik.Windows.Tests/App.config (hunk 22 above)
Comment (ddubuc) [reply Tests/Armonik.Windows.Tests/App.config]: Tests removed

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj [hunk 26]
```diff
@@ -1,37 +1,24 @@
...
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Common\ArmoniK.DevelopmentKit.Common\ArmoniK.DevelopmentKit.Common.csproj" />
+    <ProjectReference Include="../../../Common/ArmoniK.DevelopmentKit.Common/ArmoniK.DevelopmentKit.Common.csproj" />
   </ItemGroup>
 
+  <Target Name="DeleteLocalCache" BeforeTargets="Pack">
```
[the remark this answers is not in the document]
Comment (ddubuc) [reply Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj:20]: this is local cleaning in nuget cache otherwise each compilation in local should increment the version to tests locally ?

File: DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/ArmoniK.DevelopmentKit.GridServer.Client.csproj [hunk 27]
```diff
@@ -1,27 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net472;net48;net5.0;net6.0</TargetFrameworks>
```
[the remark this answers is not in the document]
Comment (ddubuc) [reply DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/ArmoniK.DevelopmentKit.GridServer.Client.csproj:4]: it's out of scope of this cleaning. NetStandard2.0 is not trivial to integrate in the project. Most of issue is with Grpc + SSlCredentials

File: Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj [hunk 28]
```diff
@@ -1,37 +1,24 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <!-- <PropertyGroup>
-    <IsWindows Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">true</IsWindows>
-    <IsLinux Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' == 'true'">true</IsLinux>
-  </PropertyGroup> -->
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net472;net48;net5.0;net6.0</TargetFrameworks>
```
[the remark this answers is not in the document]
Comment (ddubuc) [reply Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj:4]: this is local cleaning in nuget cache otherwise each compilation in local should increment the version to tests locally ?

File: Tests/Armonik.Windows.Tests/Armonik.Windows.Tests.csproj [hunk 29]
```diff
@@ -0,0 +1,81 @@
...
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
```
[the remark this answers is not in the document]
Comment (ddubuc) [reply Tests/Armonik.Windows.Tests/Armonik.Windows.Tests.csproj]: It was the default generation of VS 2022. Test has been removed

File: SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi.Client/ArmoniK.DevelopmentKit.SymphonyApi.Client.csproj [hunk 30]
```diff
@@ -1,29 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net472;net48;net5.0;net6.0</TargetFrameworks>
```
[the remark this answers is not in the document]
Comment (ddubuc) [reply SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi.Client/ArmoniK.DevelopmentKit.SymphonyApi.Client.csproj:4]: This is out of scope of this cleaning. The .net Standard 2.0 is not trivial to integrate in the project. Mostly due to Grpc + SSLCredentials and X509CryptoGraph

File: WorkerApi/tests/WorkerApiTests/Directory.Build.props (hunk 6 above)
Comment (dbrasseur-aneo) [reply WorkerApi/tests/WorkerApiTests/Directory.Build.props:3]: Unresolved

**Review by dbrasseur-aneo — APPROVED** [review]

File: DataSynapseApi/ArmoniK.DevelopmentKit.GridServer/ArmoniK.DevelopmentKit.GridServer.csproj [hunk 31]
```diff
@@ -1,24 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net6.0</TargetFrameworks>
-    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
-    <DockerfileContext>..\..\..\..\..\..\..</DockerfileContext>
     <OutputType>Library</OutputType>
-    <Authors>$(Authors)</Authors>
```
[the remark this answers is not in the document]
Comment (ddubuc) [reply DataSynapseApi/ArmoniK.DevelopmentKit.GridServer/ArmoniK.DevelopmentKit.GridServer.csproj:8]: Moved in each project

File: WorkerApi/tests/WorkerApiTests/Directory.Build.props (hunk 6 above)
Comment (ddubuc) [reply WorkerApi/tests/WorkerApiTests/Directory.Build.props]: Tests removed. Now it is :)

File: Directory.Build.props [hunk 32]
```diff
@@ -0,0 +1,89 @@
...
+    <BaseIntermediateOutputPath>$(RootProjectDir)/build/$(OSBuilder)/$(MSBuildProjectName)/obj/</BaseIntermediateOutputPath>
+    <DefaultItemExcludes>$(DefaultItemExcludes);build/**;$(RootProjectDir)/build/**;bin/**;obj/**</DefaultItemExcludes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>ANEO</Authors>
+    <Company>ANEO</Company>
+    <Product>ArmoniK</Product>
+    <Description>ArmoniK Package SDK for high level API</Description>
+    <Copyright>Copyright © ANEO 2022</Copyright>
```
Comment (lemaitre-aneo) [inline Directory.Build.props:36]:
```suggestion
    <Copyright>Copyright © ANEO 2021-2022</Copyright>
```

File: WorkerApi/tests/WorkerApiTests/UnitTest1.cs [hunk 33]
```diff
@@ -1,82 +0,0 @@
-// This file is part of the ArmoniK project
```
Comment (lemaitre-aneo) [inline WorkerApi/tests/WorkerApiTests/UnitTest1.cs:1]: Wasn't this test used?

**Review by lemaitre-aneo — COMMENTED** [review]
I think you could add the svg version of the logo instead of the png.

File: WorkerApi/tests/WorkerApiTests/UnitTest1.cs (hunk 33 above)
Comment (aneojgurhem) [reply WorkerApi/tests/WorkerApiTests/UnitTest1.cs:1]: It was not used in pipelines. I tried to run it but it did not work

Comment (ddubuc) [reply WorkerApi/tests/WorkerApiTests/UnitTest1.cs:1]: Wasn't used. And new unit test will come after test documentation
